zest/releaser/postrelease.py

```diff
@@ -22,6 +22,8 @@
     'dev_version_template': 'Template for development version number',
     'development_marker': 'String to be appended to version after postrelease',
     'new_version': 'New version, without development marker (so 1.1)',
+    'update_history': 'Should zest.releaser update the history file?',
+    'update_version': 'Should zest.releaser update the version file?',
```

Comment:
We would still be asking the user for a version, with a suggestion from us, and this would end up in the data, ready for future plugins to use it. In the towncrier command I use that version from the data as well.
But I will remove this `update_version` part. Let's keep it restricted to just the `update_history` part that I really need.